soh/src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -628,7 +627,7 @@ void EnBox_Update(Actor* thisx, PlayState* play) {
     }
 
     if (((!IS_RANDO && ((this->dyna.actor.params >> 5 & 0x7F) == 0x7C)) ||
-        (IS_RANDO && ABS(sItem.getItemId) == RG_ICE_TRAP)) &&
+        (IS_RANDO && this->getItemEntry.getItemId == RG_ICE_TRAP)) &&
```

Comment:
do you know why we used to have `ABS` here?

Reply:
the static `sItem` was what was handed off to player, with the getItemId negated (which allows the player to interact with the chest)

Now this just uses the `this->getItemEntry` I added later on for Chest Style Matches Contents.

Reply:
See https://github.com/HarbourMasters/Shipwright/blob/bb6fef6bf5aa1188cc0ac2386ed639d248455bbd/soh/src/overlays/actors/ovl_En_Box/z_en_box.c#L507-L512

Reply:
In case you're wondering, we _should_ be able to just replace what's left of `sItem` with `this->getItemEntry`, but this is all going away anyway so I erred on the side of safety and just left most of it untouched.